app-backend/datamodel/src/main/scala/Annotation.scala

```diff
@@ -352,3 +355,117 @@ object StacLabelItemProperties {
       value: Double
   )
 }
+
+@JsonCodec
+final case class AnnotationWithClasses(
+    id: UUID,
+    projectId: UUID,
+    createdAt: Timestamp,
+    createdBy: String,
+    modifiedAt: Timestamp,
+    owner: String,
+    description: Option[String],
+    machineGenerated: Option[Boolean],
+    confidence: Option[Float],
+    quality: Option[AnnotationQuality],
+    geometry: Option[Projected[Geometry]],
+    annotationGroup: UUID,
+    labeledBy: Option[String],
+    verifiedBy: Option[String],
+    projectLayerId: UUID,
+    taskId: Option[UUID],
+    classes: Json
+) extends GeoJSONSerializable[AnnotationWithClasses.GeoJSON] {
+  def toGeoJSONFeature = AnnotationWithClasses.GeoJSON(
+    this.id,
+    this.geometry,
+    AnnotationWithClassesProperties(
+      this.projectId,
+      this.createdAt,
+      this.createdBy,
+      this.modifiedAt,
+      this.owner,
+      this.description,
+      this.machineGenerated,
+      this.confidence,
+      this.quality,
+      this.annotationGroup,
+      this.labeledBy,
+      this.verifiedBy,
+      this.projectLayerId,
+      this.taskId,
+      this.classes
+    )
+  )
+}
+
+object AnnotationWithClasses {
+  final case class GeoJSON(
+      id: UUID,
+      geometry: Option[Projected[Geometry]],
+      properties: AnnotationWithClassesProperties,
+      _type: String = "Feature"
+  ) extends GeoJSONFeature
+
+  object GeoJSON {
+    implicit val annoWithClassesGeojonEncoder: Encoder[GeoJSON] =
+      Encoder.forProduct4("id", "geometry", "properties", "type")(
+        geojson => (geojson.id, geojson.geometry, geojson.properties, geojson._type)
+      )
+  }
+}
+
+final case class AnnotationWithClassesProperties(
+  projectId: UUID,
+  createdAt: Timestamp,
+  createdBy: String,
+  modifiedAt: Timestamp,
+  owner: String,
+  description: Option[String],
+  machineGenerated: Option[Boolean],
+  confidence: Option[Float],
+  quality: Option[AnnotationQuality],
+  annotationGroup: UUID,
+  labeledBy: Option[String],
+  verifiedBy: Option[String],
+  projectLayerId: UUID,
+  taskId: Option[UUID],
+  classes: Json
```

Comment:
Similar to below - is it possible to make this an actual type rather than generic `Json` -- every time we've used `Json` we end up regretting it later when the API gives us a response that is valid `Json` but not something we expected.